RELEASE-NOTES.rst

```diff
@@ -18,6 +18,293 @@
+   for patron by barcode retrieves multiple results, if the barcode
+   contains dashes.
+-  Updates the total results display to ElasticSearch ``7.9.0``.
+
+User management
+~~~~~~~~~~~~~~~
+
+-  Moves the user personal data from the user resource (JSON of the user
+   module) to the RERO ID profile (the user profile database).
+-  Extends the user resource with the following fields for patron:
+
+   -  Notes (displayed in the patron profile).
+   -  Expiration date (displayed in the patron profile).
+   -  Library affiliation.
+
+-  Group the patron data of the user in a nested structure.
+-  Adds a new Invenio account login view REST API.
+-  Moves the link between the patron record and the user profile from
+   the email to the id.
+-  Synchronizes the patron record and RERO ID profile data in both
+   directions.
+-  Sets the default user password as the birth date.
+-  Renames the patron API endpoints from ``patrons`` to ``rero_users``.
+-  Adds a user web API to return the number of patrons giver a username
+   in order to ensure that the usernames are unique.
+
+Tests
+~~~~~
+
+-  Adds fixtures data for a new organisation for testing purpose.
+   Existing records, such as organisation, library, patron type, etc.,
+   makes the writing process of Cypress tests much easier.
+-  Adds a model of a Cypress test to ease further the creation of
+   Cypress tests and to provide a list of good practices.
+-  Replaces ``cy.wait()`` by timeouts or by waiting for aliases to
+   harden Cypress tests robustness.
+-  Tests with Cypress the `circulation scenario A`_.
+-  Tests with Cypress the `circulation scenario B`_.
+-  Moves from Travis CI to GitHub actions to improve the preformance of
+   running tests at each pull request or merged commit.
+-  Updates Cypress tests to the patron module refactoring (renamed
+   ``users``).
+
+Angular application (Professional interface, search)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Initializes the application according to angular standards.
+
+Instance
+~~~~~~~~
+
+-  Updates Cypress to ``v4.12.1``.
+-  Updates ElasticSearch to ``7.9.0``.
+-  Updates Invenio to ``3.3``.
+-  Updates Celery to ``5.0.0``
+-  Improves ElasticSearch monitoring by fixing ElasticSearch duplicate
+   records computation.
+-  Fixes an issue that prevented the Celery configuration fixture to be
+   found by setting the constraint on the celery python package version
+   (``<5.0.0``).
+-  Fixes ``LXML`` errors during deployment. See `this Invenio pull
+   request`_.
+-  Enables ``invenio-admin`` and ``invenio-userprofiles``.
+-  Adds an API to display the database connection counts. It allows to
+   monitor the DB usage and to have statistics in order to decide how to
+   improve performance of deployed instance.
+
+Issues
+~~~~~~
+
+-  `rero-ils#83`_: Types are deprecated in ElasticSearch, then
+   ``document_type`` parameter should not be uses anymore. Fixed by
+   upgrading ElasticSearch to ``3.3``.
```

Comment:
Funny: a few lines above it's stated *Updates ElasticSearch to* **7.9.0**

Reply:
You're right! Should be Invenio 3.3.